src/commands/advanced_generate.ts

```diff
@@ -287,6 +297,7 @@ export default class extends Command {
         const ti_raw = ctx.interaction.options.getString("textual_inversion") ?? ctx.client.config.advanced_generate.default?.tis
         const hires_fix = ctx.interaction.options.getBoolean("hires_fix") ?? ctx.client.config.advanced_generate.default?.hires_fix ?? false
         const qr_code_url = ctx.interaction.options.getString("qr_code_url")
```

Comment:
Don't hard code defaults, if none are given in the config the horde default should be applied

Reply:
Hires-fix does default to false (it's not even supported on SDXL payloads), but I'll change it to undefined instead.